### PR TITLE
Expand the country list for the native app banner

### PR DESCRIPTION
**Pull request by phil303**
Expand the list to every country but the following: Brazil, Germany, France, Indonesia, Iran, Cuba, Myanmar, and Sudan.

:eyeglasses: @nramadas, it looks like you were the last person to touch this code.

### CODE REVIEW

**Comment by nramadas** [comment]
I think we're supposed to include all countries *except* for Brazil, Germany, France, Indonesia, Iran, Cuba, Myanmar, and Sudan.

**Comment by uzi** [comment]
:point_up: Yes, that. :nail_care:

**Comment by phil303** [comment]
lol 
1308831460_cd698b82b4

**Comment by phil303** [comment]
:haircut:

File: src/lib/smartBannerState.es6.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ const ANDROID_USER_AGENTS = [
 
 const ALLOWED_DEVICES = IOS_USER_AGENTS.concat(ANDROID_USER_AGENTS);
 
-const ALLOWED_COUNTRIES = new Set(['US', 'GB', 'AU', 'CA']);
+const DISALLOWED_COUNTRIES = new Set(['BR', 'DE', 'FR', 'ID', 'IR', 'CU', 'MM', 'SD']);
```
Comment (uzi) [inline src/lib/smartBannerState.es6.js]: :no_bell: Might be good to put the list in alphabetical order

**Comment by uzi** [comment]
:fish: optional minor nit

**Comment by uzi** [comment]
:tropical_fish:

**Comment by nramadas** [comment]
:fish:

**Pull request merged**